crates/aptos-dkg/src/pvss/chunky/chunked_elgamal.rs

```diff
@@ -215,3 +215,107 @@ impl<'a, E: Pairing> sigma_protocol::Trait<E> for Homomorphism<'a, E> {
         b"APTOS_CHUNKED_ELGAMAL_SIGMA_PROTOCOL_DST".to_vec()
     }
 }
+
+#[allow(dead_code)] // Will be used in the new PVSS
```

Comment:
Thanks for the clarifying note! 🫡

Reply:
Comments would help here btw: you never explain how this correlates the randomness